content/cli/orch_cli/deployments.md

```diff
@@ -500,6 +500,192 @@ Delete labels from a specific deployment.
 labels associated with this key will be deleted from the deployment
 
 
+### schedule
+
+A deployment schedule, a.k.a. execution schedule, allows the user to set a 
+scheduling rule for the execution of a particular workflow, such as install 
+or uninstall, on a deployment. This allows both single-use, i.e. postponed 
+executions, and recurrent executions (e.g. running each Sunday at 19:30).
+
+
+#### schedule list
+
+##### Usage
+
+`cfy deployments schedule list [OPTIONS] [DEPLOYMENT_ID]` 
+
+List all deployment schedules on the manager. If `DEPLOYMENT_ID` is provided, list only schedules of this deployment.
+
+##### Optional flags
+All the regular optional flags available for `list` commands, in addition to:
```

Comment:
Who are "All the regular optional flags available for `list` commands"? You should either specify them or provide a reference to a section that specifies them.